src/schema/sources.ts

```diff
@@ -1312,6 +1313,9 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
         (builder) => {
           builder.queryBuilder
             .andWhere(filter)
+            .andWhere(
+              `(${builder.alias}.flags->'publicThreshold')::boolean IS TRUE`,
+            )
```

Comment:
But this is on sources right?
Do we only use it for squads now?

Reply:
Yes, though let's maybe add this filter only when `filterOpenSquads` is present.